ffxivbot/handlers/QQGroupCommand_hunt.py

```diff
@@ -0,0 +1,455 @@
+from django.utils.datetime_safe import datetime
+from ffxivbot.handlers.QQUtils import *
+from ffxivbot.models import *
+
+
+def QQGroupCommand_hunt(*args, **kwargs):
+    action_list = []
+    try:
+        global_config = kwargs["global_config"]
+        group = kwargs["group"]
+        user_info = kwargs["user_info"]
+        QQ_BASE_URL = global_config["QQ_BASE_URL"]
+        action_list = []
+        receive = kwargs["receive"]
+        user_id = receive["user_id"]
+        group_id = receive["group_id"]
+        msg = ""
+
+        receive_msg = receive["message"].replace("/hunt", "", 1).strip()
+        param_segs = receive_msg.split(" ")
+        TIMEFORMAT_MDHMS = "%m-%d %H:%M:%S"
+        TIMEFORMAT_YMDHMS = "%Y-%m-%d %H:%M:%S"
+
+        try:
+            # 检测是否狩猎组群组，并且获取群组信息
+            group_info = HunterGroupID.objects.get(groupid=group_id)
+            while "" in param_segs:
+                param_segs.remove("")
+            try:
+                optype = param_segs[0].strip()
+            except IndexError:
+                optype = "help"
+
+            if (optype == "help"):
+                msg = "獭獭の狩猎时钟 alpha.1\n/hunt help：帮助\n/hunt check：查询相关\n/hunt kill：设置击杀时间相关\n/hunt edit：手动修改相关\n/hunt list：列出相关\n/hunt maintained：设置为维护后状态（建议维护结束后立即使用）"
+            elif (optype == "check"):
+                try:
+                    monster = param_segs[1].strip()
+                    try:
+                        # 获取怪物信息
+                        monster_info = MonsterList.objects.get(monstername=monster)
+                        try:
+                            # 获取怪物在各个服务器的击杀时间
+                            ktinfo = ServersMonsterKillTime.objects.get(monsterid=monster_info.monsterid)
+                            if group_info.servermark == "myc":
```

Comment:
I don't think it's a good idea to hard code the servers in code, it's already stored in the database, so the logic should be related to the database as well.